CMakeLists.txt

```diff
@@ -790,6 +790,7 @@ install(
   FILES "${CMAKE_CURRENT_BINARY_DIR}/CHANGELOG.md"
         "${CMAKE_CURRENT_SOURCE_DIR}/LICENSE"
         "${CMAKE_CURRENT_SOURCE_DIR}/VAST.spdx"
+        "${CMAKE_CURRENT_SOURCE_DIR}/python/vast-python.spdx"
```

Comment:
This will also have to be changed to it's attached to releases. Please grep for all occurrences of VAST.spdx.